weave/trace_server/interface/builtin_object_classes/llm_model.py

```diff
@@ -0,0 +1,32 @@
+from enum import Enum
+from typing import Optional, Union
+
+from pydantic import BaseModel
+
+from weave.trace_server.interface.builtin_object_classes import base_object_def
+
+
+class ModelMode(str, Enum):
```

Comment:
For here and everywhere else, I would recommend being a bit more verbose in naming: `LLMProviderModelMode` - this will avoid name conflicts in the future